src/payment/data/reducers.js

```diff
@@ -5,6 +5,7 @@ import { reducer as coupon } from '../coupon';
 
 export const basketInitialState = {
   loading: false,
+  products: [],
```

Comment:
This was necessary because the rest of the code expects it to be an empty array at least, and it's always returned as an empty array from endpoints.  This prevents us from needing downstream isArray checks.